cmake/modules/FindDEAL_II_KOKKOS.cmake

```diff
@@ -50,6 +50,25 @@ else()
       set(DEAL_II_VECTORIZATION_WIDTH_IN_BITS 0)
       # Require lambda support to use Kokkos as a backend
       KOKKOS_CHECK(OPTIONS CUDA_LAMBDA)
+
+      # Disable a bunch of annoying warnings generated by boost, template code,
+      # and in other random places:
+      #
+      # integer conversion resulted in a change of sign:
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=68")
+      # variable "i" was set but never used:
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=550")
+      # loop is not reachable:
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=128")
+      # warning #186-D: pointless comparison of unsigned integer with zero
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=186")
+      # warning #177-D: variable "n_max_face_orientations" was declared but never referenced
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=177")
+      # warning #284-D: NULL reference is not allowed
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=284")
+      # extra parentheses:
+      enable_if_supported(DEAL_II_CXX_FLAGS "-Wno-parentheses")
```

Comment:
This doesn't fit the pattern. Shouldn't this one be dealt with elsewhere (with the other warning flags for gcc/clang)?

Reply:
Yes, but these warnings are inside bundled boost but they only happen with nvcc not with gcc. I am not quite sure why, maybe the system header logic doesn't work correctly for that?

Reply:
https://github.com/dealii/dealii/commit/336c66f297c9601302006fc6be6d2d445c461f36 adds that flag when building bundled TBB in `bundled/setup_bundled.cmake`. Maybe, just do the same for bundled `Boost`?

Reply:
Thanks, @masterleinad ! I missed ./bundled/ and only checked ./cmake/ for existing functionality around the warning. Fixed.